content/src/content/jcr_root/apps/core/wcm/sandbox/components/teaser/v1/teaser/teaser.html

```diff
@@ -17,17 +17,13 @@
      data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
      data-sly-test.hasContent="${teaser.imageResource}"
      class="cmp-teaser${!wcmmode.disabled ? ' cq-dd-image' : ''}">
-
     <div class="cmp-teaser__image" data-sly-resource="${teaser.imageResource}"></div>
-    <sly data-sly-test.withCTA="${teaser.withCTA}" data-sly-list="${teaser.CTAs}">
-        <a class="cmp-teaser__call-to-action-link" href="${item.path}">${item.title}</a>
-    </sly>
-
-    <h2 class="cmp-teaser__title" data-sly-test.title="${teaser.title}">
-        <a class="cmp-teaser__title-link" href="${teaser.linkURL}" data-sly-unwrap="${!teaser.linkURL || withCTA || teaser.hideTitleLink}">${title}</a>
+    <h2 class="cmp-teaser__title" data-sly-test.title="${teaser.title}" data-sly-element="${teaser.titleType}">
+        <a class="cmp-teaser__title-link" href="${teaser.linkURL}" data-sly-unwrap="${!teaser.linkURL || teaser.hideTitleLink}">${title}</a>
     </h2>
-    <p class="cmp-teaser__description" data-sly-test.description="${teaser.description}">
-        <a class="cmp-teaser__description-link" href="${teaser.linkURL}" data-sly-unwrap="${!teaser.linkURL || withCTA || teaser.hideDescriptionLink}">${description}</a>
-    </p>
+    <div class="cmp-teaser__description" data-sly-test.description="${teaser.description}">${teaser.description @ context="html"}</div>
+    <div class="cmp-teaser__call-to-action-container" data-sly-test.withCTA="${teaser.withCTA}" data-sly-list="${teaser.CTAs}">
```

Comment:
I find this class name a little verbose. Proposal:
`cmp-teaser__actions`
`cmp-teaser__action`